src_tests/unittests/osm_configurator/model/project/test_active_project.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+import shutil
+
+from src.osm_configurator.model.project.active_project import ActiveProject
+from src.osm_configurator.model.project.config_phase_enum import ConfigPhase
+from pathlib import Path
+
+
+class MyTestCase(unittest.TestCase):
+    def test_build(self):
+        path: Path = Path("C:")
+        self.active_project: ActiveProject = ActiveProject(path, True, "TestProject1", "Das sollte funktionieren")
+        self.active_project.set_last_step(ConfigPhase.CATEGORY_CONFIG_PHASE)
+        self.active_project.get_project_saver().save_project()
```

Comment:
Isn't an assertion missing here?

Reply:
Also I don't like that you are executing a test on my C: folder, Hopfully you don't delete everything